shells/nushell/Portfile

```diff
@@ -0,0 +1,704 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        nushell nushell 0.32.0
+revision            0
+categories          shells
+platforms           darwin
+license             MIT
+maintainers         {banst @b4nst} openmaintainer
```

Comment:
sorry, one more - this line is actually not correct; your e-mail address should be listed as ```example.org:user```

Reply:
also, please change your commit message as I mentioned above

Reply:
Oops sorry I didn't catch the message about the commit. Will do